faqs/gtn/gtn_minor_correction.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Making a minor correction to any training material
+box_type: tip
+layout: faq
+contributors: [philreeddata, nomadscientist]
+area: contributors
+---
+
+If you find a minor mistake in any GTN training material, we encourage you to propose a correction. For small changes such as typos, this can be done from the browser. If you can implement the corrections yourself, in the context of where they happen, this saves a lot of time for the editors.  When you submit your suggestion, it will be carefully checked by an expert, so do not worry about breaking anything! 
+
+Outline of the steps:
+
+1. Start from the page with the minor mistake.  
+2. Open the page in the GitHub editor.  
+3. Make the correction.  
+4. Save the changes with a description of what you did.  
+5. Send the proposal to the GTN team to check, then apply the change.
+
+In this example, we will show how to correct a typo in the metadata of a learning pathway. Note, this specific typo has now been corrected.
+
+### 1. Start from the page with the minor mistake
+
+![Learning pathways page](images/minor-fix-01.png "Example of a page with a typo, this time in the page metadata")
```

Comment:
```suggestion
![Learning pathways page]({% link faqs/gtn/images/minor-fix-01.png %} "Example of a page with a typo, this time in the page metadata")
```

please use the link tag with the full path to the image as above (do same for the other images below)